employees/views.py

```diff
@@ -54,6 +56,8 @@ class MonthNavigationMixin(ContextMixin):
 
     def _get_url_from_date(self, date: datetime.date, pk: int = None) -> str:
         url_kwargs = {"year": date.year, "month": date.month}
+        if self.kwargs.keys().__contains__("project_pk"):
```

Comment:
Why not simply `if "project_pk" in self.kwargs:`?

Reply:
Done